### DISCUSSION TITLE
JK DZ11 active balencer

### DISCUSSION THREAD

**Discussion by RichGorrod** [mr-manuel/venus-os_dbus-serialbattery#42] [General]
Hello,
I have added a DZ11 balancer to a DIY seplos 16s 280aH battery pack. There is a Can/ RS485 connection on this. balancers. - Would it be possible to read this on the cerbo / node red.

Any pointers would be greatly appreciated

Below in the serial data for this unit.

[JK-B1A24S-RS485-protocol-english (1).pdf](https://github.com/user-attachments/files/16052339/JK-B1A24S-RS485-protocol-english.1.pdf)

.Many thanks.

Rich

**Comment by mr-manuel** [comment]
Duplicate https://github.com/Louisvdw/dbus-serialbattery/discussions/1090.
